src/cmd/connect.go

```diff
@@ -50,7 +50,7 @@ var (
 				tunnel, err = c.Connect(ctx, target)
 			}
 			if err != nil {
-				spinner.Fatalf(err, lang.CmdConnectErrService, err.Error())
+				return fmt.Errorf("unable to connect to the service: %w", err)
```

Comment:
```suggestion
				return fmt.Errorf("lost connection to the service: %w", err)
```
IK it was already like this but IMO this makes more sense

Reply:
I agree, my plan was to avoid changing error messages for now but this one is small.